template/_templates/component/new/component.ejs

```diff
@@ -1,57 +1,14 @@
 ---
-to: src/components/<%= name %>/component.tsx
+to: src/components/<%= h.inflection.transform(name, ['undasherize', 'classify']) %>/<%= h.inflection.transform(name, ['undasherize', 'classify']) %>.tsx
 ---
-<% const comp = name -%>
-<% if(componentType === 'Class') { -%>
-import styled from '@emotion/native';
-import React, { Component } from 'react';
-import { View, ViewStyle } from 'react-native';
-import { colors } from '../../styles';
+<% const formattedName = h.inflection.transform(name, ['undasherize', 'classify']) -%>
+import React, { SFC } from 'react';
```

Comment:
Should this be FC instead of SFC? I saw when working earlier today that SFC is deprecated and should be FC